app/viewmodels/journeys/taskList/PrepopTradeJourneyStatusesViewModel.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.journeys.taskList
+
+import controllers.journeys.income
+import models._
+import models.common.JourneyStatus.CannotStartYet
+import models.common.{BusinessId, JourneyStatus, TaxYear, TradingName, TypeOfBusiness}
+import models.journeys.Journey
+import models.journeys.Journey._
+import models.requests.TradesJourneyStatuses
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{SummaryList, SummaryListRow, Value}
+import viewmodels.govuk.summarylist._
+import viewmodels.journeys.{SummaryListCYA, determineJourneyStartOrCyaUrl, getJourneyStatus}
+
+case class PrepopTradeJourneyStatusesViewModel(tradingName: TradingName,
+                                               typeOfBusiness: TypeOfBusiness,
```

Comment:
This is the same as TradeJourneyStatusesViewModel, you could just put buildPrepopSummaryList in that existing object and reuse the methods and delete this class